src/common/src/session_config/mod.rs

```diff
@@ -836,6 +845,11 @@ impl ConfigMap {
                 name: StandardConformingStrings::entry_name().to_lowercase(),
                 setting: self.standard_conforming_strings.to_string(),
                 description: String::from("Unused in RisingWave"),
+            },
+            VariableInfo{
+                name: StreamingRateLimit::entry_name().to_lowercase(),
+                setting: self.streaming_rate_limit.to_string(),
+                description: String::from("Set streaming rate limit (rows per second)"),
```

Comment:
It looks like this parameter actually acts on a single `Executor` instance (parallelism), right?

```suggestion
                description: String::from("Set streaming rate limit (rows per second) for each parallelism"),
```

Reply:
Yes that's correct.